apps/champions/priv/skills.json

```diff
@@ -20,35 +1663,339 @@
+    "effects": [
+      {
+        "type": {
+          "duration": 5,
+          "period": 1
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifiers": [
+          {
+            "attribute": "speed",
+            "modifier_operation": "Multiply",
+            "magnitude_calc_type": "Float",
+            "float_magnitude": 1.07
+          }
+        ],
+        "executions": [
+          {
+            "type": "Heal",
+            "attack_ratio": 0.17,
+            "energy_recharge": 50,
+            "delay": 0
+          }
+        ],
+        "target_strategy": "all",
+        "target_count": 6,
+        "target_allies": true
+      }
+    ]
+  },
+  {
+    "name": "Valtimer Basic",
+    "energy_regen": 100,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+          "type": "instant"
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifier": [],
         "executions": [
           {
             "type": "DealDamage",
-            "attack_ratio": 3,
+            "attack_ratio": 1,
             "energy_recharge": 50,
             "delay": 0
           }
         ],
         "target_strategy": "random",
+        "target_count": 1,
+        "target_allies": false
+      }
+    ],
+    "cooldown": 5
+  },
+  {
+    "name": "Valtimer Ultimate",
+    "energy_regen": 0,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+          "period": 1
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifiers": [],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 3.26,
+            "energy_recharge": 50,
+            "delay": 0
+          },
+          {
+            "type": "AddEnergy",
+            "amount": -200
+          }
+        ],
+        "target_strategy": {
+          "highest": "Attack"
+        },
+        "target_count": 1,
+        "target_allies": false
+      }
+    ]
+  },
+  {
+    "name": "H4ck Basic",
+    "energy_regen": 100,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+          "type": "instant"
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifier": [],
+        "executions": [
+          {
+            "type": "DealDamage",
+            "attack_ratio": 1,
+            "energy_recharge": 50,
+            "delay": 0
+          }
+        ],
+        "target_strategy": "backline",
+        "target_count": 1,
+        "target_allies": false
+      }
+    ],
+    "cooldown": 5
+  },
+  {
+    "name": "H4ck Ultimate",
+    "energy_regen": 0,
+    "animation_duration": 0,
+    "animation_trigger": 0,
+    "effects": [
+      {
+        "type": {
+          "duration": 2,
+          "period": 1
+        },
+        "initial_delay": 0,
+        "components": [],
+        "modifiers": [
```

Comment:
Let's not handle paralyze-stun effects via speed, we should create a tag for this instead

Reply:
You did it correctly with Muflus!